build.sh

```diff
@@ -198,7 +198,7 @@ if [ "$PIPELINE" = 'build' ]; then
         --build-arg CLIENT_APPID="$CLIENT_APPID" \
         --build-arg STEAM_LOGIN="$STEAM_LOGIN" \
         --build-arg CACHE_KEY="$GAME_VERSION" \
-        -t "$GAME_IMAGE" \
+        --tag "$GAME_IMAGE" \
```

Comment:
this add unnecessary verbosity. the other params are spelt out because they lack shorthands. the commands should preferably be interactively friendly on the command-line

Reply:
see my other comment